libse/SubtitleFormats/Chk.cs

```diff
@@ -143,7 +143,7 @@ private string GetText(byte[] buffer, int start, int end)
                 if (end - start > 0)
                     text = _codePage.GetString(buffer, start, end - start);
             }
-            if (text.Length > 4 && text[0] == 0x1f && text[1] == 'R' && text[4] == '.' && "0123456789".Contains(text[2]) && "0123456789".Contains(text[3]))
+            if (text.Length > 4 && text[0] == 0x1f && text[1] == 'R' && text[4] == '.' && char.IsDigit(text[2]) && char.IsDigit(text[3]))

```

Comment:
`char.IsDigit` and `"0123456789".Contains` are different,
```
PS> [char]::IsDigit(([char]0x660))
True
```
U+0660 is Arabic-Indic digit zero.